test/fuzzing/hb-fuzzer.hh

```diff
@@ -1,4 +1,4 @@
 #include <hb.h>
 #include <stddef.h>
 
-extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size);
```

Comment:
Please don't change the signature here.  It must match what libFuzzer expects.  Add a cast to main.cc.